### DISCUSSION TITLE
Dismiss icon!

### DISCUSSION THREAD

**Discussion by danedwardsdeveloper** [tailwindlabs/heroicons#1237] [Ideas]
Hello, I found this quite funny... the icon I'm looking for is right here but I don't know what it's called!

<img width="538" alt="Screenshot 2024-10-25 at 09 55 26" src="https://github.com/user-attachments/assets/159415c0-59a9-41ea-9ae9-cb2e9ad7a8bb">

Love your icons though, many thanks!
Dan

**Comment by ndri** [comment]
It's `x-circle`, isn't it?
